### PR TITLE
patch combined_kreports.xml

### PR DESCRIPTION
**Pull request by mthang**
FOR CONTRIBUTOR:
* [x] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] License permits unrestricted use (educational + commercial)
* [ ] This PR adds a new tool or tool collection
* [x] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

The header section of the combine_kreports.xml output file embedded the galaxy directory and It affects the file parsing in the downstream. Therefore, I made the simple patch by updating this wrapper. I reported this https://github.com/galaxyproject/tools-iuc/issues/6081#issue-2350355548 . 

It would be great if the wrapper gets updated soon.

Many thanks

### CODE REVIEW

File: tools/krakentools/combine_kreports.xml [hunk 1]
```diff
@@ -35,6 +49,12 @@ combine_kreports.py
             <param name="display_headers" value="--no-headers"/>
             <param name="only_combined" value="--only-combined"/>
             <output name="output" file="only_combined_count.tabular"/>
+    	</test>
+        <test>
+            <param name="reports" value="beta_kreport_1.tabular,beta_kreport_3.tabular"/>
+            <param name="display_headers" value="--display-headers"/>
+            <param name="only_combined" value="--only-combined"/>
+            <output name="output" file="combined_kreport_w_headers.txt"/>
```
Comment (bgruening) [inline tools/krakentools/combine_kreports.xml:57]: this file is really large, can we use asserts here or only compare a subset of the file...

Comment (mthang) [reply tools/krakentools/combine_kreports.xml]: Done. I just updated the test unit 3 by adding assert_contents.

Comment (mthang) [reply tools/krakentools/combine_kreports.xml]: I just checked the error from the linting result and noticed the following.
 
   GALAXY_FORK: galaxyproject
    GALAXY_BRANCH: release_24.1
    MAX_CHUNKS: 4
    MAX_FILE_SIZE: 1M

does it mean that the total file size in test-data folder can't exceed 1M and that's reason to use asserts? The test-data folder of the krakentools is 6.7M in total and it's way more than 1M.

Comment (mthang) [reply tools/krakentools/combine_kreports.xml]: I just fixed this tool by using the "asserts" in the test unit.

**Review by bgruening — APPROVED** [review]

**Comment by mvdbeek** [comment]
Attention: deployment skipped!

https://github.com/galaxyproject/tools-iuc/actions/runs/10198209622

**Comment by bgruening** [comment]
Deployed.

**Pull request merged**